tests/modules/net_vpc/examples/factory.yaml

```diff
@@ -31,7 +31,7 @@ values:
     project: my-project
     region: europe-west1
     role: null
-    secondary_ip_range:
+    secondary_ip_ranges:
```

Comment:
this part should be ok, no need to update any of the files in this folder.

Reply:
Fixed.